package.json

```diff
@@ -16,9 +16,9 @@
   "bugs": "https://github.com/mozilla/fxa-auth-mailer/issues",
   "license": "MPL-2.0",
   "dependencies": {
-    "bluebird": "2.9.34",
+    "bluebird": "3.0.6",
```

Comment:
Is this the bluebird version of death that caused a memory-leak in the auth-server?  It may be worth holding off on upgrading that across the board until we understand that's going on there (unless it's a security update of course).